src/logger/Logger.py

```diff
@@ -0,0 +1,39 @@
+import datetime
+import os
+from enum import Enum
+
+class Severity(Enum):
+    DEBUG = 1
+    INFO = 2
+    WARNING = 3
+    ERROR = 4
+
+class Logger:
+    
+    def __init__(self, debug=False, filename='log.txt'):
+        self.debug = debug
+        self.filename = filename
+        self._f = open(self.filename,'a+')
+
+        self._f.write('[{}]  {}  {} \n'.format( 'INFO', str(datetime.datetime.now()), 'Logger started.'))
+
+
+    def log(self, message, severity=Severity.INFO):
+        timestamp = datetime.datetime.now()
+
+        if severity == Severity.ERROR:
+            self._f.write('[{}]  {}  {} \n'.format('ERROR', str(timestamp), message))
```

Comment:
Minor nitpick but you should just leave the 'ERROR' string literal inside the string instead of putting it in format, if nothing else it looks better that way. Also format should automatically convert timestamp to string so you don't need to use str().